include/elasticlient/client.h

```diff
@@ -51,7 +51,8 @@ class Client {
      * \param timeout      Elastic node connection timeout.
      */
     explicit Client(const std::vector<std::string> &hostUrlList,
-                    std::int32_t timeout = 6000);
+                    std::int32_t timeout = 6000,
+                    const std::initializer_list<std::pair<const std::string, std::string>>& proxyUrlList = {});
```

Comment:
A few notes:
  * to maintain API/ABI stability, it is recommended to add another constructor ([link](https://community.kde.org/Policies/Binary_Compatibility_Issues_With_C%2B%2B)) instead of adding parameters
  * parameter was missing in the doc string
  * another option would be a setter method, as the initializer_list would probably block adding another parameters to such constructor